browser/src/control/Control.UIManager.js

```diff
@@ -445,13 +448,13 @@ L.Control.UIManager = L.Control.extend({
 					this.map.sidebar.setupTargetDeck('.uno:MasterSlidesPanel');
 				} else if (this.getBooleanDocTypePref('NavigatorDeck', false)) {
 					app.socket.sendMessage('uno .uno:SidebarShow');
-					app.socket.sendMessage('uno .uno:Navigator');
-					this.map.sidebar.setupTargetDeck('.uno:Navigator');
+					//app.socket.sendMessage('uno .uno:Navigator');
```

Comment:
Possibly we want to completely remove Navigator option from sidebar context here. To not try to close other decks in sidebar.
We should have separate setting for "left-sidebar". Similar to this one.